packages/core/react/src/__tests__/WalletProviderMobile-test.tsx

```diff
@@ -110,6 +114,12 @@ describe('WalletProvider when the environment is `MOBILE_WEB`', () => {
     beforeEach(() => {
         localStorage.clear();
         jest.clearAllMocks().resetModules();
+        rpcEndpoint = 'https://api.mainnet-beta.solana.com';
+        jest.mocked(useConnection).mockImplementation(() => ({
+            connection: {
+                rpcEndpoint,
+            } as Connection,
+        }));
```

Comment:
Worth testing the case when there's no ConnectionProvider?

Reply:
Maybe? The default implementation of the `ConnectionProvider` when not supplied is `{}` which will produce `connection?.rpcEndpoint === undefined`, so I'm not too worried.